ci/infrastructure/scripts/infrastructure.sh

```
@@ -707,7 +710,7 @@ findDynamicPorts() {
       if [ "$FREE" = "" ]; then
         #echo " - netstat says $THIS_PORT is free - using it"
 	      eval "VS_CONTAINER_EXT_PORT_"$VS_CONTAINER_SERVICE"="$THIS_PORT
```

Shellcheck (0.10.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)